me.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>module one Capstone project.</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="social">
+      <a href="#"><img src="./images/facebook.png"  width ="10" height="10px" alt="logo"></a>
+      <a href="#"><img src="./images/twitter.png"  width ="10" height="10px" alt="logo"></a>
+      <a class="social-items" href="#">English</a>
+      <a class="social-items"  href="#">My page</a>
+      <a class="social-items"  href="#">logout</a>
+    </div>
+    <div class="nav-item-desk">
+     <a  href="index.html"><img src="./images/ralph.jpg"  width ="78" height="30px" alt="logo"></a>
+      <div class="nav-items">
+        <a class="nav-link1" href="me.html">About</a>
+      <a class="nav-link" href="#program">Programs</a>
+      <a class="nav-link" href="#contactt">Join</a>
+      <a class="nav-link" href="#contactt">Sponsor</a>
+      <a class="nav-link" href="#contactt">News</a>
+      <a class="nav-link6" href="#contactt">CC Campaign</a>
+    </div>
+    </div>
+    <div id="nav-bar">
+      <div class="nav-item">
+          <a class="nav-link" href="#about">About</a>
+          <a class="nav-link" href="#program">Programs</a>
+          <a class="nav-link" href="#contactt">Join</a>
+          <a class="nav-link" href="#contactt">Sponsor</a>
+          <a class="nav-link" href="#contactt">News</a>
+          <a class="nav-link" href="#contactt">CC Campaign</a>
+      </div>
+      <div id="ham" class="hamburger">
+        <div class="hamburger-bar"></div>
+        <div class="hamburger-bar"></div>
+        <div class="hamburger-bar"></div>
+      </div>
```

Comment:
- [ optional ] For Mobile screen size I recommend you to add "Home" page link at the "About " page to redirected to "Home" page which helps to improve user interaction.

![Capture](https://user-images.githubusercontent.com/93598189/192116190-0b6f6fb6-5d95-419e-ae60-36b0a9518542.PNG)